marklogic-junit5/examples/simple-ml-gradle/README.md

```diff
@@ -39,7 +39,7 @@ Next, add the following dependencies - this assumes you're using Gradle version
     dependencies {
       // existing dependencies
       
-      testCompile "com.marklogic:marklogic-junit:0.10.0"
+      testCompile "com.marklogic:marklogic-junit5:1.0.beta"
```

Comment:
Same issue. Shouldn't publish with dependency on a beta release, use a final release

Reply:
Fixed in Pr #101